.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
-  - repo: https://github.com/ambv/black
-    rev: stable
```

Comment:
This seems to be the recommended approach based on: https://pre-commit.com/#using-the-latest-version-for-a-repository